internal/painter/gl/painter.go

```diff
@@ -275,8 +275,5 @@ func (p *painter) textureScale(v float32) float32 {
 func NewPainter(c fyne.Canvas, ctx driver.WithContext) Painter {
 	p := &painter{canvas: c, contextProvider: ctx}
 	p.SetFrameBufferScale(1.0)
-
-	p.glInit()
```

Comment:
I may be missing something, but in this context it looks like some init code isn't being called any more - is this intended?

Reply:
If “in this context” means “at this point of time”, yes. The code in question is executed later with `p.Init()`. The painter is useless unless `Init()` has been called anyhow, so no harm done. OTOH, the error logging in `glInit()` depends on the painter’s context been initialized which is done consistently in `p.Init()` for all implementations. At least for “mobile” it is not possible to do so earlier.

Reply:
See also: 1fd952073c893b5fae88a0abf9eed845a3ce1ae1

Reply:
Thanks for the explanation, sounds like it's sorted :)